xbmc/playlists/SmartPlayList.cpp

```diff
@@ -821,6 +821,7 @@ std::string CSmartPlaylistRule::GetVideoResolutionQuery(const std::string &param
 
   switch (m_operator)
   {
+    using enum SearchOperator;
```

Comment:
unnecessary with using enum CDatabaseQueryRule::SearchOperator at the top of the file.

Reply:
well spotted. thanks.